### DISCUSSION TITLE
迅维网 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#6] [模板讨论]
迅维网 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
Failed at 6/11 request,
Fail assert: {"re": "错误", "from": "content"} from failed_asserts,
Request URL: https://www.chinafix.com/member.php?mod=logging&action=login&loginsubmit=yes&handlekey=login&loginhash={{loginhash|urlencode}}&inajax=1

**Reply by wjf0214** [reply]
一般的是OCR验证码识别错误导致的，如果觉得错误多的话，可以使用cookie版本。

**Comment by aixu521** [comment]
2024-07-29 17:30:15
日志：Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.chinafix.com/plugin.php?id=k_misign%3Asign

**Comment by kissgaby** [comment]
失效了,不知道有没有大哥,更新一下!

**Comment by miaoxiaozhao** [comment]
Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.chinafix.com/plugin.php?id=k_misign%3Asign

**Comment by miaoxiaozhao** [comment]
失效了，cookie版也失效了。
